src/Components/GameBoard.js

```diff
@@ -365,7 +366,49 @@ class GameBoard extends Component {
     // TODO: LEARN HOW TO DO API CALL HERE LOL
     let response = await fetch(url);
     let newBoard = await response.json();
-    this.setState({ board: newBoard, loading: false});
+    this.setState({ board: newBoard, loading: false, turn: newBoard['turn']});
+
+    let made_graph = create_graph(this.state.board['graph'])
+    //console.log(made_graph);
+    this.setState({ graph: made_graph});
+
+
+    //console.log(newBoard)
+  }
+
+    playCard2 = async () => {
```

Comment:
Need to abstract the 3 separate functions that do the same thing into one function.